sushi-config.yaml

```diff
@@ -8,11 +8,11 @@ canonical: http://hl7.org/fhir/uv/cgm
 name: Continuous Glucose Monitoring
 title: Continuous Glucose Monitoring
 description: Communicate Continuous Glucose Monitoring (CGM) data using FHIR
-status: draft # draft | active | retired | unknown
-version: 0.1.0
+status: active # draft | active | retired | unknown
+version: 1.0.0-ballot
 fhirVersion: 4.0.1 # https://www.hl7.org/fhir/valueset-FHIR-version.html
-copyrightYear: 2024+
-releaseLabel: ci-build # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
+copyrightYear: 2025+
+releaseLabel: STU 1 Ballot # ci-build | draft | qa-preview | ballot | trial-use | release | update | normative+trial-use
```

Comment:
I think this should be one of the documented values. That is, `ballot`, not STU 1 Ballot.

Reply:
These are what have been agreed to by FMG in the past, and they will ultimately decide what these values are.